test/helpers/proxy/proxy.go

```diff
@@ -27,14 +28,18 @@ var (
 	sampleNamespaceNetworkPolicy = path.Join(project.TestDataDir(), "network/sample-ns-denial.yaml")
 
 	proxyDeploymentPath = path.Join(project.TestDataDir(), "network/proxy.yaml")
+	proxySCCPath        = path.Join(project.TestDataDir(), "network/proxy-scc.yaml")
 
 	ProxySpec = &dynatracev1beta1.DynaKubeProxy{
-		Value: "http://squid.proxy:3128",
+		Value: "http://squid.proxy.svc.cluster.local:3128",
 	}
 )
 
 func SetupProxyWithTeardown(builder *features.FeatureBuilder, testDynakube dynatracev1beta1.DynaKube) {
 	if testDynakube.Spec.Proxy != nil {
+		if kubeobjects.ResolvePlatformFromEnv() == kubeobjects.Openshift {
```

Comment:
Maybe a own function that is just called here would make sense to keep this method clean